index.html

```diff
@@ -1,33 +1,61 @@
 <!DOCTYPE html>
 <html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+        <link rel="icon" type="image/x-icon" href="/assets/alchemy-favicon.png" />
+        
+        <link rel="preconnect" href="https://fonts.googleapis.com" />
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+        <link
+            rel="stylesheet"
+            href="https://fonts.googleapis.com/css2?family=Rubik:wght@300;400;600&family=Titillium+Web:wght@200;400;600&display=swap"
+        />
+        <link rel="stylesheet" href="/styles/reset.css" />
+        <link rel="stylesheet" href="/styles/global.css" />
+        <title>Lucky the Dog </title>
+    </head>
 
-<head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <link rel="icon" type="image/x-icon" href="/assets/alchemy-favicon.png" />
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link rel="stylesheet"
-        href="https://fonts.googleapis.com/css2?family=Rubik:wght@300;400;600&family=Titillium+Web:wght@200;400;600&display=swap">
-    <link rel="stylesheet" href="/styles/reset.css" />
-    <link rel="stylesheet" href="/styles/global.css" />
-    <title>Web App</title>
-</head>
+    <body>
+        <header>
+            <img src="/assets/html_css_js.png" />
+            <h1>Lucky Shanks</h1>
+        </header>
 
-<body>
-    <header>
-        <img src="/assets/html_css_js.png">
-        <h1>Web App</h1>
-    </header>
+        <main></main>
+        <h1> All about Lucky
+        </h1>
+        <p> Lucky is a male 13 year old husky mix moving rug. He was originally a hurricane rescue dog from the Midwest but now lives in the Pacific Northwest with his mom and stepdad. 
+            <ul class="container">
 
-    <main>
+                    <img src=/assets/lucky_the_dog.png alt="Look at that good boy">; 
+                    <li>He's a good dog
+                    </li> 
+              
+                    <img src=/assets/lucky_snow.png alt="It feels so good on his winter coat!">
+                    <li> He loves to lay down in the snow
 
-    </main>
+                    </li>;
```

Comment:
You don't want the semicolons here, that's a javascript thing not an HTML thing.

Also try and have consistency on the nesting, you'll want it more like:
'<li>
   He's a good dog
</li>'